test/components/Timeline/Playbook/DisplayGlobalActionsTest.js

```diff
@@ -0,0 +1,152 @@
+'use strict'
+import { expect } from 'chai'
+import React from 'react'
+import moment from 'moment'
+import createComponent from '../../../helpers/shallowRenderHelper'
+import {
+    DisplayGlobalActions,
+    DisplayGlobalAction,
+    mapStateToDisplayGlobalActionsProps
+} from '../../../../src/components/Timeline/Playbook/DisplayGlobalActions'
+import Play from '../../../../src/components/Timeline/Playbook/Play'
+
+describe('DisplayGlobalActions', function () {
+    describe('Component - Display All', function () {
+        it('Should output a div with a list of DisplayGlobalAction children when passed actions', function () {
+            const input = {
+                ticketId: 1,
+                engagementId: 2,
+                incidentId: 3,
+                actions: [
+                    {
+                        name: 'testOne'
+                    },
+                    {
+                        name: 'testTwo'
+                    },
+                    {
+                        name: 'testThree'
+                    }
+                ]
+            }
+
+            const testResult = createComponent(DisplayGlobalActions, input)
+
+            expect(testResult.type).to.equal('div')
+            expect(testResult.props.children[0].type).to.equal('div')
+            expect(testResult.props.children[2].type).to.equal('div')
+            expect(testResult.props.children[3]).to.not.exist
+        })
+
+        it('Should output an empty div when passed no actions', function () {
+            const input = {
+                ticketId: 1,
+                engagementId: 2,
+                incidentId: 3
+            }
+
+            const testResult = createComponent(DisplayGlobalActions, input)
+
+            expect(testResult.type).to.equal('div')
+            expect(testResult.props.children).to.not.exist
+        })
+    })
+
+    describe('Component - Display Single', function () {
+        const testAction = {
+            name: 'test'
+        }
+        it('Should output a div with a span child, a line break, and a Play', function () {
```

Comment:
I'm going to assume 'because that's how we want it to look' is insufficient. What's a good way to describe the desired display of UI elements, and how much of that should be included in the test spec?